packages/uui-base/lib/mixins/LabelMixin.ts

```diff
@@ -37,8 +37,8 @@ export const LabelMixin = <T extends Constructor<LitElement>>(
     @property({ type: String })
     public label!: string;
 
-    connectedCallback() {
-      super.connectedCallback();
+    firstUpdated(_changedProperties: Map<string | number | symbol, unknown>) {
+      super.firstUpdated(_changedProperties);
```

Comment:
If I recall correctly, `firstUpdated` does not require a call to `super`.

```suggestion
    firstUpdated() {
```

Reply:
Note: I reverted this change again elsewhere, because it is used in a mixin, and we don't know what the superClass is necessarily. That means we have to call `super.firstUpdated` as well to make sure the chain works.